File: backend/src/main/java/com/team05/petmeeting/domain/shelter/dto/ShelterCommand.java
package com.team05.petmeeting.domain.shelter.dto;

import java.time.LocalDateTime;

public record ShelterCommand(
    String careRegNo,
    String careNm,
    String careTel,
    String careAddr,
    String careOwnerNm,
    String orgNm,
    LocalDateTime updTm
) {
}

File: backend/src/main/java/com/team05/petmeeting/domain/shelter/dto/ShelterCommand.kt
package com.team05.petmeeting.domain.shelter.dto

import java.time.LocalDateTime

data class ShelterCommand(
    val careRegNo: String,
    val careNm: String?,
    val careTel: String?,
    val careAddr: String?,
    val careOwnerNm: String?,
    val orgNm: String?,
    val updTm: LocalDateTime
)
